scripts/nci_env_deatest/datacube_config.conf

```diff
@@ -0,0 +1,12 @@
+[datacube]
+db_hostname: agdcdev-db.nci.org.au
+db_port: 6432
+db_database: ncitest_datacube_db
+db_username: sm9911
```

Comment:
We probably don't need to specify the username here, to make it more easily run-able by others.

Reply:
Thanks for your feedback, Damien. 
I will make changes to the scripts as mentioned, to make it more generic and useable.
Also, I will add a README.md file and remove the username from the config file.